src/components/common/CourseReviewBox.tsx

```diff
@@ -241,9 +252,11 @@ const CourseReviewBoxContent = ({
   const notifyUpdate = () => toast(REVIEW_SUCCESS.updated);
 
   const profId =
-    selectedProf === -1 || selectedProf === profsTeaching.length
+    selectedProf === -1 || selectedProf === 0
```

Comment:
Can we pull the `0` into a const at the top called `const NO_PROFESSOR_INDEX = 0`